### DISCUSSION TITLE
21 days waiting for sponsorship approval

### DISCUSSION THREAD

**Discussion by duran004** [community/community#114442] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I know, I know, I know. The busier they are, the later we'll get approval. BLA BLA BLA . 
I would be grateful if someone in authority could explain why I did not receive approval after thinking about your classic answers. The fact that I cannot get approval in my country while the citizen of the European country can get approval quickly, and that there is no phone code of my country in the two-factor verification, no sms is sent ...etc. it FEELS LIKE RACISM to me

**Comment by DHayes10** [comment, marked as answer]
Hi @duran004, 

We appreciate you reaching out regarding your profile. Your listing has been forwarded for approval. Your account should reflect the update within the next hour. Our apologies for the delay as we have recently had more volume. 

Thanks, 
GitHub Sponsors
